test/Microsoft.AspNet.Mvc.TagHelpers.Test/ScriptTagHelperTest.cs

```diff
@@ -88,28 +85,6 @@ public static TheoryData RunsWhenRequiredAttributesArePresent_Data
                     {
                         new TagHelperAttributeList
                         {
-                            ["asp-src-include"] = "*.js"
-                        },
-                        tagHelper =>
-                        {
-                            tagHelper.SrcInclude = "*.js";
-                        }
```

Comment:
This removal is fine because `RenderScriptTags_WithFileVersion()` covers the case.